http/analysis-service/jobs/run.sh

```diff
@@ -48,6 +48,7 @@ if [ "$(jq -r '.num_workers|type' < $JOB_CONFIG)" == "number" ]; then # Spark cl
         --applications Name=Spark Name=Hive \
         --bootstrap-actions Path=s3://${EMR_BUCKET}/bootstrap/telemetry.sh,Args=\["--timeout","$TIMEOUT"\] \
         --configurations https://s3-${REGION}.amazonaws.com/${EMR_BUCKET}/configuration/configuration.json \
+        --log-uri s3://telemetry-public-analysis-2/frank \
```

Comment:
Let's make this `--log-uri "s3://$DATA_BUCKET/$JOB_NAME/logs"`, then users will be able to view the logs directly from atmo as well.